packages/core/admin/admin/src/content-manager/pages/EditView/Information/index.js

```diff
@@ -1,25 +1,40 @@
 import React, { useRef } from 'react';
+import propTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 import { useCMEditViewDataManager } from '@strapi/helper-plugin';
-import { Box } from '@strapi/design-system/Box';
-import { Divider } from '@strapi/design-system/Divider';
-import { Typography } from '@strapi/design-system/Typography';
-import { Flex } from '@strapi/design-system/Flex';
-import { Stack } from '@strapi/design-system/Stack';
+import { Box, Divider, Flex, Stack, Typography } from '@strapi/design-system';
+
 import { getTrad } from '../../../utils';
 import getUnits from './utils/getUnits';
 import { getFullName } from '../../../../utils';
 
-const Informations = () => {
+const KeyValuePair = ({ label, value }) => {
+  return (
+    <Flex justifyContent="space-between" as="p">
```

Comment:
what about https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd & https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt 👀 feels _very_ appropriate?

Reply:
It does and I thought about it, however: I have no idea how to replicate that with our DS (because `dt` and `dd` do not have a wrapper). Let me play with it a bit.

Reply:
This is the closest I can get it to:

```js
const Dt = styled(Flex)`
  flex-grow: 0;
`;

const Dd = styled(Flex)`
  align-self: flex-end;
  flex-grow: 0;
`;

const KeyValuePair = ({ label, value }) => {
  return (
    <>
      <Dt as="dt">
        <Typography fontWeight="bold" textColor="neutral600">
          {label}
        </Typography>
      </Dt>

      <Dd as="dd">
        <Typography>{value}</Typography>
      </Dd>
    </>
  );
};
```

<img width="247" alt="Screenshot 2023-01-20 at 12 21 25" src="https://user-images.githubusercontent.com/2244375/213683649-bf6e8930-7822-4f95-b671-605b12a872cb.png">

Do you have an idea how they would not wrap?

Reply:
Think you'd need to use forwardAs on the original Flex item. This is why I prefer to use a different prop e.g. tag to target what 'as' should mean 👀

Reply:
The problem is not in the generated markup (at least you can ignore that for now), but in how to build that layout using DS components. I really don't know how to use `Flex` or `Box` to come up with that layout without using floats (value) and a clear: left on the label ...

Reply:
I'll check this out on Monday I've done it a few times 🤓

Reply:
Done in https://github.com/strapi/strapi/pull/15523